pkg/sync/v5/data/Service/default/router.yaml

```diff
@@ -14,6 +14,8 @@ spec:
     port: 80
   - name: 443-tcp
     port: 443
+  - name: router-stats
```

Comment:
you need to fix up service.beta.kubernetes.io/azure-dns-label-name above - that should go to where you're creating the new IP

Reply:
Added